src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
The variable name might not be meaningful enough, maybe change to "deadline"

Reply:
i will consider it!